app/assets/v2/css/search.css

```diff
@@ -0,0 +1,170 @@
+:root {
```

Comment:
var defined as root by not used and seems only related to search not root, maybe something for colors.css instead ?